### PR TITLE
Add gpu tag filter

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

File: .github/workflows/wave.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ env:
   # renovate: datasource=github-releases depName=seqeralabs/wave-cli versioning=semver
   WAVE_VERSION: "1.5.0"
   # renovate: datasource=github-releases depName=askimed/nf-test versioning=semver
-  NFTEST_VERSION: "0.9.1"
+  NFT_VER: "0.9.1"
```
Comment (sateeshperi) [inline .github/workflows/wave.yml:16]:
```suggestion
  NFT_VER: "0.9.2"
```

File: .github/actions/nf-test-action/action.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ runs:
     - name: Install nf-test
       uses: nf-core/setup-nf-test@v1
       with:
-        version: "${{ env.NFTEST_VERSION }}"
+        version: "${{ env.NFT_VER }}"
```
Comment (edmundmiller) [inline .github/actions/nf-test-action/action.yml:38]: https://github.com/nf-core/modules/pull/7019

Is `NFT_VER` special? `NXF_VER` as well I couldn't remember in that PR but renovate wanted `_VERSION` by default.

Comment (mashehu) [reply .github/actions/nf-test-action/action.yml:38]: NXF_VER is nextlfow standard: https://www.nextflow.io/docs/latest/install.html#updates

**Review by edmundmiller — APPROVED** [review]
LGTM. I like the seperation of the GPU tests, made this really easy to review.

**Review by maxulysse — APPROVED** [review]

**Pull request merged**